lib/hsa/mcwamp_hsa.cpp

```diff
@@ -787,13 +787,17 @@ namespace Kalmar {
 // HSAQueue is the implementation of accelerator_view for HSA back-and.  HSAQueue
 // points to RocrQueue, or to nullptr if the HSAQueue is not currently attached to a RocrQueue.
 struct RocrQueue {
+    static void callbackQueue(hsa_status_t status, hsa_queue_t* queue, void *data) {
+        STATUS_CHECK(status, __LINE__);
```

Comment:
Could you described how we could distinguish different abort conditions (i.e. vm fault, user abort(), etc..)?